vars/sonarExecuteScan.groovy

```diff
@@ -124,19 +124,21 @@ void call(Map parameters = [:]) {
         def worker = { config ->
             try {
                 withSonarQubeEnv(config.instance) {
+                    List envVars = []
 
-                        loadSonarScanner(config)
+                    loadSonarScanner(config)
 
-                        if(fileExists('.certificates/cacerts')){
-                            sh 'mv .certificates/cacerts .sonar-scanner/jre/lib/security/cacerts'
-                        }
-
-                        if(config.organization) config.options.add("sonar.organization=${config.organization}")
-                        if(config.projectVersion) config.options.add("sonar.projectVersion=${config.projectVersion}")
-                        // prefix options
-                        config.options = config.options.collect { it.startsWith('-D') ? it : "-D${it}" }
+                    if(fileExists('.certificates/cacerts')){
+                        envVars = envVars.plus("SONAR_SCANNER_OPTS=-Djavax.net.ssl.trustStore=${env.WORKSPACE}/.certificates/cacerts")
```

Comment:
@CCFenner still you should cater for potential whitespaces in `env.WORKSPACE` as we learned from @andre2007. Besides that looks fine and less hacky than the previous approach 👍.